src/main/java/frc/robot/Constants.java

```diff
@@ -126,34 +129,50 @@
     public static final NeutralMode PIVOT_NEUTRALMODE = NeutralMode.Coast;
     public static final double PIVOT_DEFAULT_SPEED = 0.2;
     public static final int PIVOT_DOWN_LIMIT = (-50200 / 100) * 81; // 45000
-    public static final int PIVOT_UP_LIMIT = (-800 / 100) * 81; // 5000 
+    public static final int PIVOT_UP_LIMIT = (-800 / 100) * 81; // 5000
     public static final int PIVOT_VELOCITY = 5000;
     public static final double PIVOT_AFFECT_GRAVITY = -0.02;
 
     public static final int PIVOT_PID_SLOT = SLOT_0;
-    // public static final MotionMagic PIVOT_MOTION_MAGIC = new MotionMagic(0.1023, 0.0, 0, 0);
+    // public static final MotionMagic PIVOT_MOTION_MAGIC = new MotionMagic(0.1023,
+    // 0.0, 0, 0);
     public static final MotionMagic PIVOT_MOTION_MAGIC = new MotionMagic(0.1023, 0.0, 0, 0);
-    // public static final MotionMagic PIVOT_MOTION_MAGIC = new MotionMagic(0.0, 0.0, 0.0, 0.0);
+    // public static final MotionMagic PIVOT_MOTION_MAGIC = new MotionMagic(0.0,
+    // 0.0, 0.0, 0.0);
 
-    // Hopper   
+    // Hopper
     public static final int HOPPER_LEFT_ID = 7;
     public static final int HOPPER_RIGHT_ID = 8;
     public static final boolean HOPPER_REVERSE = false;
     public static final NeutralMode HOPPER_NEUTRALMODE = NeutralMode.Coast;
     public static final double HOPPER_LEFT_SPEED = 0.85;
     public static final double HOPPER_RIGHT_SPEED = 0.7;
-    
+
+    // ================================================================================
+    // PathWeaver Data
+    // ================================================================================
+
+    public static final String DRIVEBASE_PATHWEAVER_PATH = "";
+    public static final DriveConfig DRIVEBASE_PATHWEAVER_CONFIG = new DriveConfig.Builder() //
```

Comment:
Are we not going to use the other constants that were in the Auto/Drive Constant files?

Reply:
This will be discussed at the meeting